### PR TITLE
feat: change some feature about `BulletY`

### PR DESCRIPTION
**Pull request by jeongwoo903**
## Major changes
- Increase speed +1
- 6 random colors
- change Stripe

### CODE REVIEW

File: src/entity/BulletY.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
  *
  */
 public class BulletY extends Entity {
-
+    Color[] colors = {Color.WHITE, Color.YELLOW, Color.GREEN, Color.MAGENTA, Color.ORANGE, Color.PINK, Color.LIGHT_GRAY};
```
Comment (highlees) [inline src/entity/BulletY.java:16]: I think it would be better to move this part into the ```setSprite``` method.
Thank you for your efforts!

Comment (ghost) [inline src/entity/BulletY.java:16]: [P3]

I think it would be better to move this part into the setSprite method.
Thank you for your efforts!

**Review by ghost — APPROVED** [review]

File: src/entity/BulletY.java [hunk 2]
```diff
@@ -46,10 +46,18 @@ public BulletY(final int positionX, final int positionY, final int speed, final
      * Sets correct sprite for the bullet, based on speed.
      */
     public final void setSprite() {
-        if (speed < 0)
+        if (speed < 0) {
+            /*
+                To apply the same color when an item is picked up.
+             */
+            Color nowColor = colors[(int) ((Math.random() * (7 - 1)) + 1)];
```
Comment (doyoon323) [inline src/entity/BulletY.java:53]: P[3] 

I think it doesn't need to set a new variable nowColor. 
Thank you!

**Review by doyoon323 — APPROVED** [review]

File: src/entity/BulletY.java (hunk 2 above)
Comment (jeongwoo903) [reply src/entity/BulletY.java:53]: That's what I thought at first, too. 
But then I realized that if you eat an item that spawns 3 bullets, each bullet spawns a different color.

저도 처음엔 그렇게 생각했었습니다. 
그런데 3개의 총알이 나가는 아이템을 먹으면 총알마다 색상이 다르게 나가는 경우가 있어서 필요하에 만들어 두었습니다 😅

**Pull request merged**